ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -44,15 +47,22 @@ tests:
     make lint
   container:
     clone: true
-    from: golangci-lint-shellcheck
+    from: test-bin
+- as: helm-lint
+  commands: |
+    export XDG_CACHE_HOME=/tmp/.cache
+    make helm-lint
+  container:
+    from: test-bin
+  optional: true
 - as: unit
   commands: make unit-test
   container:
-    from: src
+    from: test-bin
 - as: verify
   commands: make verify
   container:
-    from: src
```

Comment:
Why do we need `test-bin` for `make verify`? Can this remain `src`?

Reply:
Same as for unit-test

Reply:
I think my comment from above also applies here.

Reply:
Moved back to `src`